__tests__/api.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const testData = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const app = require("../app");
+
+beforeEach(() => {
+  console.log("seeding!");
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET: /api/topics", () => {
+  test("200: should return all list of topics along with the 200 status code", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.topics.length).toBe(3);
+        const testArr = res.body.topics;
```

Comment:
why not initiailse variable above assertion on line 22 and save yourself some key strokes ?